### DISCUSSION TITLE
Is it possible to remove hyperlinks with the app?

### DISCUSSION THREAD

**Discussion by bagajohny** [pdfarranger/pdfarranger#1027] [Q&A] [answered]
I want to know if it is possible to remove the hyperlinks from a PDF page with the use of pdfarranger app. I have come across a pdf which has marked an entire page as hyperlink so no matter where I click on the page, it opens the link in web browser. Due to this I am unable to select and copy any text from the page. Please help me solve the issue. I will link the concerned PDF below. Check the second page of the pdf file for the hyperlink.
[BL_Mumbai_08-01-2024.pdf](https://github.com/pdfarranger/pdfarranger/files/13857188/BL_Mumbai_08-01-2024.pdf)

**Comment by dreua** [comment]
No, this is not possible with PDF Arranger. You could try Libreoffice Draw, maybe the link is an object you can easily delete. There are other libraries and apps for extracting text from PDFs but I can't give free support for this.

**Reply by bagajohny** [reply]
Is this something you can consider implementing in the PDF Arranger app?

**Reply by jeromerobert** [reply, marked as answer]
No. PDF Arranger manipulate each PDF page as black box. It never work inside pages, so it can not do what you are asking for.
